src/Common/tests/System/Collections/TestingTypes.cs

```diff
@@ -147,6 +147,9 @@ public int GetHashCode(int x)
 
     #region TestClasses
 
+#if netstandard17
+    [Serializable]
+#endif
```

Comment:
Looks like you missed this one when cleaning up the #ifdefs around Serializable.